corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -120,19 +125,33 @@ hqDefine('hqwebapp/js/multiselect_utils', [
                 this.search_left.cache();
                 // remove search option so that user doesn't get confused
                 this.search_right.val('').search('');
-                $('#' + removeAllId).removeClass('disabled').prop('disabled', false);
+                if (!disableModifyAllActions) {
+                    $('#' + removeAllId).removeClass('disabled').prop('disabled', false);
+                }
                 this.search_right.cache();
             },
             afterDeselect: function () {
                 // remove search option so that user doesn't get confused
                 this.search_left.val('').search('');
-                $('#' + selectAllId).removeClass('disabled').prop('disabled', false);
+                if (!disableModifyAllActions) {
+                    $('#' + selectAllId).removeClass('disabled').prop('disabled', false);
+                }
                 this.search_left.cache();
                 this.search_right.cache();
             },
         });
 
+        multiselect_utils.rebuildMultiselect = function (elementId, multiselectProperties) {
+            var $element = _.isString(elementOrId) ? $('#' + elementOrId) : $(elementOrId);
```

Comment:
Where is `elementOrId` coming from?

Reply:
Good call. I lucked out in that this function was defined within the `createFullMultiselectWidget` widget included the same `elementOrId`. However this made me realize there shouldn't be any reason to define `rebuildMultiselect` within this function as it is a standalone method on a multiselect element. 

See fbc681ac645d9d1d5a2048db5f7d17aceaf32802